src/Fixture/Factory/SearchableFixtureFactory.php

```diff
@@ -67,6 +67,8 @@ protected function configureOptions(OptionsResolver $resolver): void
                 ->setNormalizer('option', LazyOption::findOneBy($this->productOptionRepository, 'code'))
             ->setDefault('filterable', false)
             ->setDefault('searchable', false)
+            ->setDefault('filterable', false)
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏remove duplicate filterable configure option

```suggestion
            ->setDefault('filterable', false)
            ->setDefault('searchable', false)
```